ios/FluentUI/Core/Colors2.swift

```diff
@@ -0,0 +1,181 @@
+    }
+    if let brandBackground1Pressed = provider.brandBackground1Pressed(for: theme)?.dynamicColor {
+        brandColors[.brandBackground1Pressed] = brandBackground1Pressed
+    }
+    if let brandBackground1Selected = provider.brandBackground1Selected(for: theme)?.dynamicColor {
+        brandColors[.brandBackground1Selected] = brandBackground1Selected
+    }
+    if let brandBackground2 = provider.brandBackground2(for: theme)?.dynamicColor {
+        brandColors[.brandBackground2] = brandBackground2
+    }
+    if let brandBackground2Pressed = provider.brandBackground2Pressed(for: theme)?.dynamicColor {
+        brandColors[.brandBackground2Pressed] = brandBackground2Pressed
+    }
+    if let brandBackground2Selected = provider.brandBackground2Selected(for: theme)?.dynamicColor {
+        brandColors[.brandBackground2Selected] = brandBackground2Selected
+    }
+    if let brandBackground3 = provider.brandBackground3(for: theme)?.dynamicColor {
+        brandColors[.brandBackground3] = brandBackground3
+    }
+    if let brandBackgroundTint = provider.brandBackgroundTint(for: theme)?.dynamicColor {
+        brandColors[.brandBackgroundTint] = brandBackgroundTint
+    }
+    if let brandBackgroundDisabled = provider.brandBackgroundDisabled(for: theme)?.dynamicColor {
+        brandColors[.brandBackgroundDisabled] = brandBackgroundDisabled
+    }
+    if let brandForeground1 = provider.brandForeground1(for: theme)?.dynamicColor {
+        brandColors[.brandForeground1] = brandForeground1
+    }
+    if let brandForeground1Pressed = provider.brandForeground1Pressed(for: theme)?.dynamicColor {
+        brandColors[.brandForeground1Pressed] = brandForeground1Pressed
+    }
+    if let brandForeground1Selected = provider.brandForeground1Selected(for: theme)?.dynamicColor {
+        brandColors[.brandForeground1Selected] = brandForeground1Selected
+    }
+    if let brandForegroundTint = provider.brandForegroundTint(for: theme)?.dynamicColor {
+        brandColors[.brandForegroundTint] = brandForegroundTint
+    }
+    if let brandForegroundDisabled1 = provider.brandForegroundDisabled1(for: theme)?.dynamicColor {
+        brandColors[.brandForegroundDisabled1] = brandForegroundDisabled1
+    }
+    if let brandForegroundDisabled2 = provider.brandForegroundDisabled2(for: theme)?.dynamicColor {
+        brandColors[.brandForegroundDisabled2] = brandForegroundDisabled2
+    }
+    if let brandStroke1 = provider.brandStroke1(for: theme)?.dynamicColor {
+        brandColors[.brandStroke1] = brandStroke1
+    }
+    if let brandStroke1Pressed = provider.brandStroke1Pressed(for: theme)?.dynamicColor {
+        brandColors[.brandStroke1Pressed] = brandStroke1Pressed
+    }
+    if let brandStroke1Selected = provider.brandStroke1Selected(for: theme)?.dynamicColor {
+        brandColors[.brandStroke1Selected] = brandStroke1Selected
+    }
+    return brandColors
+}
+
+// MARK: Colors
+
+public enum BrandColorsForOverriding: CaseIterable {
```

Comment:
1. Does this need to be public?
2. If so, does it need an `@objc` name?

Reply:
Good catch, I don't think it needs to be public!